templates/create-daml-app/ui/package.json.template

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@daml.js/__PROJECT_NAME__": "file:../daml.js/__PROJECT_NAME__-0.1.0",
+    "@daml.js/__PROJECT_NAME__": "file:daml.js/__PROJECT_NAME__-0.1.0",
```

Comment:
Does `../daml.js` not work with npm or why did you change this?

Reply:
No, we need the generated code in the root level of the ui package.